bedevere/prtype.py

```diff
@@ -0,0 +1,46 @@
+"""Label a pull request based on its type."""
```

Comment:
I'm struggling with whether this needs to be in its own file or should be moved to `files`. Up until this point every thematic thing that Bedevere did was contained in a single file, but `files` actually spreads across 3 at this point. I'm thinking it might be better to move this into `files` and then either do the same for `news.check_news()` or at least leave a comment as to why that one check breaks the overall rule.

Reply:
Yes, I wasn't sure how far I should go with restructuring this.  One reason I thought of having this in its own module is if we were able to add the other type labels by looking at the bug tracker.  So, for example, another routing entry and function could be added here to retrieve the bpo info and classify to `type-enhancement` or `type-bugfix` if the bug tracker had that as the type.  That lookup wouldn't make sense in the `file(path)` module and would start getting more like the Ni bot looking for the CLA.